### PR TITLE
(TRX): contract address modal don't show if no url available

### PR DESCRIPTION
**Pull request by LFBarreto**
Updated contract address modal to not show if no url available.

### Type

UI Polish

### Context

Trx trc10 tokens

### Parts of the app affected / Test plan

Tokens contract info modal

### CODE REVIEW

File: src/screens/Account/AccountHeaderRight.js [hunk 1]
```diff
@@ -39,6 +45,15 @@ const AccountHeaderRight = ({ navigation, account, parentAccount }: Props) => {
...
+
+    const url = getAccountContractExplorer(
+      explorerView,
+      account,
+      parentAccount,
+    );
+
     return (
       <>
         <Touchable event="ShowContractAddress" onPress={toggleModal}>
```
Comment (gre) [inline src/screens/Account/AccountHeaderRight.js:59]: mmh, i might be wrong, but shoudln't we return null completely if there is no url? that's what i meant by "not showing it at all if it's not possible"

Comment (LFBarreto) [reply src/screens/Account/AccountHeaderRight.js:59]: we have a modal that asks to hide the token as well we have to be able to hide the token except the modal showing the contract address link won't show :+1:

Comment (gre) [reply src/screens/Account/AccountHeaderRight.js:59]: ok

**Comment by gre** [comment]
Also we are not supposed to conditionally render a modal to make it appear. It's better to use the isOpened Boolean for that. Am I seeing something wrong here? cc @juan-cortes

**Comment by juan-cortes** [comment]
The first modal appearing or not was fixed on the other pr I did, as to conditionally rendering them or not, it seemed cleaner this way but I haven't done mobile in a while maybe you're right

**Pull request merged**